includes/components/class-admin.php

```diff
@@ -49,6 +49,9 @@ public function __construct( $args = [] ) {
 		// Share usage data.
 		add_action( 'hivepress/v1/events/weekly', [ $this, 'share_usage_data' ] );
 
+        // Update license key.
+        add_action( 'update_option_hp_hivepress_license_key', [ $this, 'update_license_key' ] );
```

Comment:
Please check what happened with WPCS because the indents are definitely off, WPCS should've corrected this on the file save. WPCS uses tabs instead of spaces, maybe this is the reason, but it should correct this and replace spaces automatically in any case